testsuite/features/secondary/min_ssh_push.feature

```diff
@@ -0,0 +1,48 @@
+# Copyright (c) 2020 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+Feature: Register a salt system to be managed via SSH push
+
+  Scenario: Delete the salt minion for ssh-reverse bootstrap
+    Given I am on the Systems overview page of this "sle_minion"
```

Comment:
We had a convention (introduced by Dario Maiocchi) to always use a minion name that reflects client's role. In this case "sle_minion" would also be called "sle_reverse_ssh_minion" or something alike.

I'm not a big fan of this convention and we could also drop it alltogether, but then we need to also remove it at the other places.

If not, we need to conform to it.

Reply:
let's keep the convention, please! But what about some shorter name? "ssh_push_minion"?

Reply:
Ok, I will go with `ssh_push_minion`

Reply:
After review doc, maybe "ssh push" is not the best way to identify this client.
See that https://documentation.suse.com/external-tree/en-us/suma/4.1/suse-manager/architecture/salt-contact-method-overview.html

![image](https://user-images.githubusercontent.com/2827771/96986530-0e6c9300-1522-11eb-9483-c6ffafbb3240.png)

![image](https://user-images.githubusercontent.com/2827771/96986732-54c1f200-1522-11eb-8571-c26051fd304b.png)

So... maybe is better to change in the feature file name and in the host name by something else like "ssh_tunnel_minion", sorry for all this confusion...